src/pages/industries/media-entertainment.jsx

```diff
@@ -0,0 +1,183 @@
+    'Cilium is designed to provide high-performance, secure, and scalable networking and observability. It offers a wide range of features, helping companies in the media and entertainment industry deliver seamless experiences, reduce operational complexity, and secure their environment. ',
+  ],
+  imageSrc: PublicSPeakingBee,
+  imageAlt: 'ebeedex public speaking bee',
+};
+
+const newyorktimesTestimonial = {
+  logo: NewYorkTimesLogo,
+  description:
+    'Cilium was a critical choice to increase networking performance while providing identity and application-aware security and visibility for cloud native workloads running on EKS at the New York Times.',
+  quotedText:
+    'Perhaps the most important feature from Hubble UI is the network flow itself. If you look deeper into the flow you’ll see full deep rich information about every single packet traversed between different services. And that’s important so that we can build understanding of how traffic is flung between services.',
+  withPerson: true,
+  name: 'Ahmed Bebars',
+  role: 'Software Engineer, Delivery Engineering, The New York Times',
+  url: 'https://cilium.io/blog/2022/10/13/publishing-user-story/',
+};
+
+const bulletSection1 = {
+  heading:
+    'Enhance Content Delivery and User Experience with Cilium’s Transparent Acceleration and  Load Balancing ',
+  withImage: true,
+  paragraphs: [
+    'Streaming platforms and content delivery services need to scale rapidly to meet the ever-growing consumer demands for high-quality and on-demand media. Cilium offers intelligent load balancing, ensuring that the high volume of user requests typical for OTT platforms are distributed efficiently. This results in consistent content and smooth streaming experiences for end users.',
+    'By leveraging technologies like XDP (eXpress Data Path), bandwidth manager, and BIG TCP, Cilium optimizes the datapath, significantly reducing latency. For an industry that thrives on seamless content delivery, this is paramount.',
+  ],
+  imageSrc: MediaImage1,
+  imageAlt: 'cilium big tcp stats',
+};
+
+const bulletSection2 = {
+  heading: 'Robust Security, Scalability, and Flexibility at a Reduced Operational Overhead',
+  paragraphs: [
+    'Safeguarding content and user data is vital in media and entertainment. Cilium enables efficient workload isolation, protecting the entire ecosystem from potential breaches. As platforms grow, Cilium adapts to shifting network needs. ',
+    "Cilium integrates with both traditional and cloud-native infrastructures. Companies like Cosmonic have extended Cilium beyond Kubernetes to platforms like Nomad. In scenarios where it is impossible to use Cilium as the Default CNI, Cilium's CNI chaining mode lets you utilize Cilium's features like Hubble for observability and Tetragon for security, ensuring organizations can benefit from Cilium regardless of their infrastructure choices.",
```

Comment:
rewrite to tie it back to the industry. The picture should be updated too since it is about performance